doc/blog/201907-SkiaSharp-for-wasm-support.md

```diff
@@ -0,0 +1,59 @@
+# SkiaSharp support for WebAssembly
+
+Skia is an open source 2D graphics library which provides common APIs that work across a variety of hardware and software platforms. It serves as the graphics engine for Google Chrome and Chrome OS, Android, Mozilla Firefox and Firefox OS, and many other products. 
+
+Today we are proud to announce that Uno now has initial support for Skia via the [Uno.SkiaSharp.Views](https://www.nuget.org/packages/Uno.SkiaSharp.Views) package which was published to NuGet moments ago. Head on over [skiasharp-wasm.platform.uno](https://skiasharp-wasm.platform.uno) to see Skia, Uno and WebAssembly working together. The WebAssembly specification is under active heavy development and thus for the best experience we recommend using a desktop computer and the latest canary version of your favourite browser. 
+
+![SkiaSharp for WebAssembly](Assets/201906-skiasharp-demo.png)
+
+
+You can find the source code for [skiasharp-wasm.platform.uno](https://skiasharp-wasm.platform.uno) over at [unoplatform/Uno.SkiaSharp](https://github.com/unoplatform/Uno.SkiaSharp). Internally this application uses [SkiaSharp](https://github.com/mono/SkiaSharp), a .NET binding to [P/Invoke the Skia API](https://github.com/mono/SkiaSharp/blob/master/binding/Binding/SkiaApi.cs#L96-L97) and a [custom Skia build](https://github.com/mono/skia) to provide the [C API to allow for the .NET interop](https://github.com/mono/skia/blob/xamarin-mobile-bindings/include/c/sk_canvas.h#L18).
+
+Inside of the `Uno.SkiaSharp.Views` package you'll find support for the [SKXamlCanvas](https://docs.microsoft.com/en-us/dotnet/api/skiasharp.views.uwp.skxamlcanvas?view=skiasharp-views-uwp-1.68.0) UWP control which enables drawing using Skia in a specific section of the XAML visual tree.
+
+You can experiment with `SKXamlCanvas` by playing with [this sample in our samples repository](https://github.com/nventive/Uno.Samples/tree/master/UI/SkiaSharpTest). Alternatively you can take it for a spin with the following XAML and Code Behind.
+
+XAML:
+```xaml
+<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <skia:SKXamlCanvas x:Name="test" PaintSurface="OnPaintSurface" />
+</Grid>
```

Comment:
```suggestion
</Grid>

Code Behind:
```csharp
private void OnPaintSurface(object sender, SKPaintSurfaceEventArgs e)
{
    // the canvas and properties
    var canvas = e.Surface.Canvas;

    // get the screen density for scaling
    var display = DisplayInformation.GetForCurrentView();
    var scale = display.LogicalDpi / 96.0f;
    var scaledSize = new SKSize(e.Info.Width / scale, e.Info.Height / scale);

    // handle the device screen density
    canvas.Scale(scale);

    // make sure the canvas is blank
    canvas.Clear(SKColors.Yellow);

    // draw some text
    var paint = new SKPaint
    {
        Color = SKColors.Black,
        IsAntialias = true,
        Style = SKPaintStyle.Fill,
        TextAlign = SKTextAlign.Center,
        TextSize = 24
    };

    var coord = new SKPoint(
        scaledSize.Width / 2, 
        (scaledSize.Height + paint.TextSize) / 2);
    canvas.DrawText("SkiaSharp", coord, paint);
}
```
```